streams/src/main/java/org/apache/kafka/streams/state/internals/WindowStoreUtils.java

```diff
@@ -32,12 +32,11 @@
     private static final int TIMESTAMP_SIZE = 8;
 
     /** Inner byte array serde used for segments */
-    public static final Serde<Bytes> INNER_KEY_SERDE = Serdes.Bytes();
-    public static final Serde<byte[]> INNER_VALUE_SERDE = Serdes.ByteArray();
-    public static final StateSerdes<Bytes, byte[]> INNER_SERDES = new StateSerdes<>("rocksDB-inner", INNER_KEY_SERDE, INNER_VALUE_SERDE);
+    static final Serde<Bytes> INNER_KEY_SERDE = Serdes.Bytes();
+    static final Serde<byte[]> INNER_VALUE_SERDE = Serdes.ByteArray();
+    static final StateSerdes<Bytes, byte[]> INNER_SERDES = new StateSerdes<>("rocksDB-inner", INNER_KEY_SERDE, INNER_VALUE_SERDE);
 
-
-    public static <K> byte[] toBinaryKey(K key, final long timestamp, final int seqnum, StateSerdes<K, ?> serdes) {
+    static <K> byte[] toBinaryKey(K key, final long timestamp, final int seqnum, StateSerdes<K, ?> serdes) {
```

Comment:
As mentioned above - perhaps change this to return `Bytes` and then we won't need to `Bytes.wrap` in various places